team-norms/retrospectives.md

```diff
@@ -16,13 +16,16 @@ The prime directive should be read before a retrospective begins.
 
 ## Retrospectives
 
-These retrospectives occur every 2 weeks on a Friday. There are 2 hours booked for this purpose. 
-
 There are two elements to retrospectives:
 
 * Retrospective (smaller groups performing retrospectives)
 * Comradrospective (whole company retrospective format)
 
+### Schedule
+
+Retrospectives should occur at least once every two weeks and should take 30-60 minutes.
```

Comment:
Academy retros have a 90 min time box but we rarely need the full 90 mins. #JustSayin'